### PR TITLE
1598 Move IntroActivity to app module

### LINKED ISSUE #1598: Move IntroActivity to app module
**Is your feature request related to a problem? Please describe.**
Custom apps should not have the intro/onboarding screen as it contains information not relevant to them that could be confusing to a user

**Describe the solution you'd like**
Move IntroActivity to app module

### PR DESCRIPTION
**Pull request by macgills**
Fixes #1598

Add intro to app module

### CODE REVIEW

File: app/src/kiwix/play/contact-website.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-http://www.kiwix.org
```
Comment (abdulwd) [inline app/src/kiwix/play/contact-website.txt]: Why were these files added and are now being removed?

Comment (macgills) [reply app/src/kiwix/play/contact-website.txt]: This was a part of an initiative of storing the play store listing in source control, it is a feature of Gradle Play Publisher, you can run the `bootstrap` command and it downloads all the assets but it has been more of a pain than anything and it seems like we should just manage it on the play store itself. 
Also they were in the wrong folder. The `kiwix` flavour/sourceSet no longer exists post modularisation so they weren't being used.

I'll admit to being fairly scatterbrained post the custom apps work and it should have been its own pr.
